DEPS

```diff
@@ -45,7 +45,7 @@ vars = {
   # Dart is: https://github.com/dart-lang/sdk/blob/main/DEPS.
   # You can use //tools/dart/create_updated_flutter_deps.py to produce
   # updated revision list of existing dependencies.
-  'dart_revision': 'e9c75accbd18418c5c647a33398ffbfddcbc7a36',
+  'dart_revision': '29d4cf8934769f9dc21f1a04b4256c86634e524e',
```

Comment:
Do we not need to also run tools/dart/create_updated_flutter_deps.py as described in the comment above?

Reply:
When I was doing https://github.com/flutter/engine/commit/49b7cd7283c622a24892bd981e3a38f8b263b0c5 (later reverted because I thought it wasn't necessary and we could modify the google3 source code instead) I did

```sh
gclient sync # get an updated Dart DEPS file to match dart_revision
tools/dart/create_updated_flutter_deps.py
```

Reply:
Yes. That would only update the LICENSES hash. In this case, there was nothing to update (which is rare).

Reply:
(Your second comment wasn't visible until I submitted)

Generally, for CPs, you can skip using the python script on release branches and manually update the revision. `Linux Unopt` will verify the DEPS file is correct (it's the fastest builder).

Reply:
Friday night brain - We did need the golden hash, but for some reason the check went green on presubmit, but failed CI. I filed https://github.com/flutter/flutter/issues/107761 and followed up with a quick fix (it shouldn't impact the fuchsia artifacts generating)